Task-57018: Hidden space indication in an event for a member . (#1452)

montassarhamadi opened the pull request:
Problem: when opening the event1 it's well indicated that it's added under a hidden space instead of display name for validation and hidden spaces.
 Fix: replace the empty value of parameter userId with the correct id of the current user in the method buildEntityFromSpace () to check if this user is a member of the space or not.

ghost commented:
### CodeSee Review Map:
[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/Meeds-io/social/1453/b641eeec/109b5e66adf061e6739f22f37f6d95685d8e4f6a.svg)](https://app.codesee.io/r/reviews?pr=1453&src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

Review [in an interactive map](https://app.codesee.io/r/reviews?pr=1453&src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

View [more CodeSee Maps](https://app.codesee.io/r/maps?src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

#### Legend
![CodeSee Map Legend](https://app.codesee.io/Legend_v2.png)

boubaker commented on webapp/portlet/src/main/webapp/vue-apps/common/components/ExoSpaceAvatar.vue:
```diff
@@ -32,7 +32,7 @@
             <p
               class="text-truncate subtitle-2 my-auto hidden-space">
-              {{ $t('spacesList.label.hiddenSpace') }}
+              {{ displayName }}
```
Since, this is a new PR for this old task, I add a remark on it.  The space name shouldn't be accessible when it's hidden and the user isn't member of, please check the property `notAccessibleSpace`. Please check with @AzmiTouil to get more information.

montassarhamadi committed:
Task-57018: Hidden space indication in an event for a member.
Problem: when opening the event1 it's well indicated that it's added under a hidden space instead of display name for validation and hidden spaces.
Fix: replace the empty value of parameter userId with the correct id of the current user  in the method buildEntityFromSpace () to check if this user is a member of the space or not.
